code/modules/status_system/statusEffects.dm

```diff
@@ -3125,12 +3125,14 @@
 
 		onAdd()
 			..()
+			for (var/datum/statusEffect/art_curse/nightmare/curse in src.owner.statusEffects)
+				return
```

Comment:
if locate in

Reply:
goddamit I knew there was a better way to do that and I just could not remember it